cmd/vicadmin/server.go

```diff
@@ -262,7 +263,7 @@ func (s *server) Authenticated(link string, handler func(http.ResponseWriter, *h
 		log.Infof("User with a valid auth cookie at %s is authenticated.", connectingAddr[0])
 		handler(w, r)
 	}
-	s.mux.Handle(link, gorillacontext.ClearHandler(http.HandlerFunc(authHandler)))
+	s.Handle(link, http.HandlerFunc(authHandler))
```

Comment:
Uhm, I think you still need to nest the `ClearHandler` because it clears out context related to the session at the end of the request, as I recall.. docs are here http://www.gorillatoolkit.org/pkg/context

Unfortunately I don't remember exactly what the side effect of not using it is, but I believe it amounts to leaking something we don't want to leak.